calico/getting-started/windows-calico/quickstart.md

```diff
@@ -376,6 +377,83 @@ The following steps install a Kubernetes cluster on a single Windows node, with
 %>
 {% endtabs %}
 
+Congratulations! You now have a Kubernetes cluster with {{site.prodnameWindows}} and a Linux control node.
+
+#### Install {{site.prodnameWindows}} using HostProcess containers
+
+With Kubernetes v1.22, a new Windows container type called "HostProcess containers" can run directly on the host with access to the host network namespace,
+storage and devices. With this feature, {{site.prodnameWindows}} can now be installed and managed using Kubernetes resources such as Daemonsets and ConfigMaps,
```

Comment:
...namespace,storage, and devices.

Reply:
I think this is a duplicate of the comment above